Homarus/src/Controller/HomarusController.php

```diff
@@ -121,8 +120,22 @@ public function convert(Request $request)
                 "frag_keyframe+empty_moov ";
         }
 
-        // Arguments to ffmpeg command are sent as a custom header
+        // Arguments to ffmpeg command are sent as a custom header.
         $args = $request->headers->get('X-Islandora-Args');
+
+        // Reject messages that try to set loglevel. We have to force
+        // it to be '-loglevel error'. Anything more verbose caues
+        // issues with large files.
+        if (preg_match("/\-loglevel/", $args)) {
```

Comment:
From the [PHP preg_match page](https://www.php.net/manual/en/function.preg-match.php)
> Tip: Do not use preg_match() if you only want to check if one string is contained in another string. Use strpos() instead as it will be faster.

So probably should be 
```
if (strpos($args, '-loglevel') !== FALSE) {
```